private/bufpkg/bufmodule/bufmodule.go

```diff
@@ -93,6 +95,14 @@ type Module interface {
 	// Documentation gets the contents of the module documentation file, buf.md and returns the string representation.
 	// This may return an empty string if the documentation file does not exist.
 	Documentation() string
+	// BreakingConfig returns the breaking change check configuration set for the module.
+	//
+	// This may be nil, since older versions of the module would not have this stored.
```

Comment:
At the least, this should be documented to have a consistent version with `LintConfig`